BSIP9999 Distributed Bitcoin Cross Claim Gateway

```diff
@@ -0,0 +1,110 @@
+
+
+BSIP: 9999
+Title: Distributed Bitcoin Cross Claim Gateway
+Authors: litepresence finitestate@tutamail.com  
+Status: Draft
+
+BSIP: <BSIP number>
+Title: <BSIP title>
+Authors: <list of authors' real names and optionally, email addrs>
+Status: [ Draft | Accepted | Installed | Voting | Deferred | Rejected | Superseded | Obsoleted ]
+
+Type: Protocol
+
+Created: 2020-09-09
+
+Discussion: <url>
+
+Abstract
+----------------
+
+polkaBTC is a full reserve distributed gateway
+
+polkaBTC "vault" is a contract that anyone can issue polkaBTC from - to any other party
+witnesses run bitcoin spv nodes and post headers
+the statistical mode of the headers are then a "BTC parachain" on the issuing chain
+issuers post collateral much like borrowing an MPA
+issuer must maintain collateral of core token on the issuing chain
+or other token on issuing chain) at stochastic market rate
+issuers can charge a fee to those who request or redeem polkaBTC
+issuer collateral ratio, fees charged, and vault size is public rpc info
+In practice, request and redeem work much like a gateway from the user level except:
+    instead of judging gateways based on perceived trustworthiness
+    you judge gateways based on fees charged
+additionally polkaBTC is equivilent regardless of which user issued it
+
+Motivation
+----------------
+
+Traditional IOU gateways require trusted third parties which have historically ended with "exit scams"
+
+Rational
+----------------
+
+A smart contract XCLAIM style gateway ensures contractual performance through collateralization
+
+Specifications
+----------------
+
+Witnesses upload bitcoin headers
+Anyone can issue "polkaBTC" to anyone by locking BTC on Bitcoin Mainnet
```

Comment:
Technically, how to achieve "locking BTC on Bitcoin Mainnet"?

Reply:
locking - in this sense - means Alice (vault client) has given Bob (vault operator) full possession of her BTC. 
Bob is now contractually obliged to maintain (BTS) collateral on the issuing chain of vaultBTC.

Reply:
further reading:

https://interlay.gitlab.io/polkabtc-spec/spec/issue.html#requestissue
https://interlay.gitlab.io/polkabtc-spec/spec/issue.html#executeissue

I'm working on a full Alice/Bob thought experiment write up of the issue/redeem functionality. 
I've probably made some misstatements, as my time is currently limited; hence "Draft".

Reply:
My reading is Bob can only issue vaultBTC by locking up enough BTS so it's sort of like creating bitBTC and then selling it for BTC.

Reply:
> Bob is now contractually obliged to maintain (BTS) collateral

This hasn't worked out well for bitAssets in (BTS-)bear marekts in the past. Why would it work better for collateralizing off-chain assets? If you want to make it "secure" you need a high collateral ratio which makes this thing super capital **in**-efficient. The more volatile your collateral is, the worse this becomes.

Reply:
in theory - you could back a "polkaBTC" vault construct with honest.btc as easily as you could with core token.


now bob wants to be a vault operator; he buys honest.btc on the open market - he's not a borrower at all
now he puts up his - owned - honest.btc as collateral 1:1 and that rate stays pegged so long as honest.btc retains 1:1 value with real btc; his loan would never require any maintenance.    the oracle of the peg could simply be the dex market price of honest.btc:bts vs the feed price of honest.btc:bts

Reply:
> My reading is Bob can only issue vaultBTC by locking up enough BTS so it's sort of like creating bitBTC and then selling it for BTC.

except that there is no way to sell bitBTC for BTC without an htlc arrangement or cex

here with polkaBTC bob has real BTC in hand after the tx and all accounting has been handled by the issuing chain (Bitshares) with awareness of bitcoin headers via a network of spv witness oracles.